### PR TITLE
Feature/dev environment test

### PR DESCRIPTION
**Pull request by gdellorusso**
changed environment variable name "api.operations.eservice.basePath" to align with other projects

### CODE REVIEW

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -20,8 +20,10 @@ jobs:
         uses: actions/checkout@v3
       - name: normalize branch name for tagging
         run: |
-          NORM_TAG=$(echo "$GITHUB_REF_NAME" | tr -s "/" "-")
-          echo "NORM_TAG=$NORM_TAG" >> $GITHUB_ENV
+          BRANCH=$GITHUB_REF_NAME
+          TAG="${BRANCH////-}"
+          echo "TAG=$TAG" >> $GITHUB_ENV
+
```
Comment (micdes-pagopa) [inline .github/workflows/build.yaml:26]: Why this change? The current version is already doing this but with a more readable syntax

File: .github/workflows/build.yaml [hunk 2]
```diff
@@ -36,4 +38,4 @@ jobs:
           file: Dockerfile
           push: true
           tags: |
-            ghcr.io/${{ github.repository }}:${{ env.NORM_TAG }}
+            ghcr.io/${{ github.repository }}:${{ env.TAG }}
```
Comment (micdes-pagopa) [inline .github/workflows/build.yaml:41]: Same as above

**Review by nttdata-rtorsoli — APPROVED** [review]

**Review by galales — APPROVED** [review]

**Review by micdes-pagopa — APPROVED** [review]
